src/configs/cmd_duration.rs

```diff
@@ -12,6 +12,7 @@ pub struct CmdDurationConfig<'a> {
     pub disabled: bool,
     pub show_notifications: bool,
     pub min_time_to_notify: i64,
+    pub notification_timeout: u64,
```

Comment:
Maybe I am missing something but can this not just be a `u32`? Then we don't need to cast it?

Reply:
I tried building as u32 but then got this error:
```
   Compiling starship v1.2.1 (/home/lsanche/Projects/starship)
error[E0599]: no method named `load_config` found for type `u32` in the current scope
  --> src/configs/cmd_duration.rs:6:17
   |
6  | #[derive(Clone, ModuleConfig, Serialize)]
   |                 ^^^^^^^^^^^^ method not found in `u32`
   |
   = help: items from traits can only be used if the trait is implemented and in scope
note: `ModuleConfig` defines an item `load_config`, perhaps you need to implement it
  --> src/config.rs:41:1
   |
41 | / pub trait ModuleConfig<'a>
42 | | where
43 | |     Self: Sized + Clone,
44 | | {
...  |
55 | |     }
56 | | }
   | |_^
   = note: this error originates in the derive macro `ModuleConfig` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0599`.
error: could not compile `starship` due to previous error
```

So I came to the assumption that module options weren't designed with u32 in mind. But I did not look very hard to see if this is indeed the case.

Reply:
Without looking further into the code, my guess is that this is because TOML requires that implementations can "losslessly" handle signed 64-bit integers, so the deserialization code will not not allow us to deserialize integers into u32s due to possible overflow. Dunno why it allows us to do u64s though...

Reply:
I think we would need to implement the ModuleConfig trait for u32 similar to [u64 here](https://github.com/starship/starship/blob/master/src/config.rs#L83).

Reply:
> I think we would need to implement the ModuleConfig trait for u32 similar to [u64 here](https://github.com/starship/starship/blob/master/src/config.rs#L89). 

I can work on this later. Should I open a separate PR for this?

Reply:
Made a commit that implements ModuleConfig for u32, removes the need to check value inside of cmd_duration.rs.